AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Internal/NativeWebSocketProvider.swift

```diff
@@ -72,10 +72,15 @@ class NativeWebSocketProvider: NSObject, AWSTranscribeStreamingWebSocketProvider
             switch result {
             case .failure(let error):
                 let status = AWSTranscribeStreamingClientConnectionStatus.closed
+                let nsError = error as NSError?
```

Comment:
Error isn't `nil`, so you should be able to do `let nsError = error as NSError`, and do away with the optional chaining. My recollection is that `Swift.Error` always successfully casts to `NSError`

Reply:
Removing `?` produces 
```
'Error?' is not convertible to 'NSError'; did you mean to use 'as!' to force downcast?
```
and force cast is also not allowed.

Reply:
if you can't, i would go with
```
if let error = error as NSError? {
   
}
```